packages/transport/src/core/TransportCoreErrors.ts

```diff
@@ -79,8 +79,46 @@ class Messages {
         );
     }
 
-    public missingOrInactiveRelationship(address: string) {
-        return new CoreError("error.transport.messages.missingOrInactiveRelationship", `An active Relationship with the given address '${address}' does not exist.`);
+    public missingOrInactiveRelationship(address: string[] | string) {
+        if (typeof address !== "string" && address.length > 1) {
+            return new CoreError(
+                "error.transport.messages.missingOrInactiveRelationship",
+                `An active Relationship with the given addresses '${address}' do not exist, so you cannot send them a Message.`
+            );
+        }
+        if (typeof address === "string") {
+            return new CoreError(
+                "error.transport.messages.missingOrInactiveRelationship",
+                `An active Relationship with the given address '${address}' does not exist, so you cannot send them a Message.`
+            );
+        }
+        return new CoreError(
+            "error.transport.messages.missingOrInactiveRelationship",
+            `An active Relationship with the given address '${address[0]}' does not exist, so you cannot send them a Message.`
+        );
+    }
+
+    public peerDeleted(address: string[]) {
```

Comment:
I'd be in favor of `deletedPeer`.

Reply:
but if we use `peerInDeletion` and `peerHasDeletionInfo` wouldn't be more consequent to use `peerDeleted`?

Reply:
Mhh `deletedPeer` seems grammatically more fitting than `peerDeleted` to me, but no strong opinion on this.